src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	Person person;
+	HashMap <Person, ArrayList<String>> mapOfTags;
```

Comment:
Since an object of `Tagging` is associated with a Person already, there is no need to put Person as key for this map. Instead, you can use a `Set` to store your tags for a person.